### PR TITLE
activity bonus, player test fails

### PR DESCRIPTION
**Pull request by santhoshnarayan**
not sure why im getting different results for g.confirmed?

all other tests pass

activity bonus only shows up after confirmation of the game, the bonus is applied to every group.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -78,7 +78,9 @@
     t.string   "unconfirmed_email"
     t.datetime "created_at"
     t.datetime "updated_at"
+    t.integer  "games_count"
```
Comment (vpontis) [inline db/schema.rb]: I don't know why this keeps being added.

File: test/models/game_test.rb [hunk 2]
```diff
@@ -78,10 +78,10 @@ def setup
     g.losers = [@c, @d]
     g.save
 
-    assert_equal 1000, @a.points
-    assert_equal 1000, @b.points
-    assert_equal 1000, @c.points 
-    assert_equal 1000, @d.points
+    assert_equal @a.points, 1000.0, "Player A's points transfered before confirmation"
```
Comment (vpontis) [inline test/models/game_test.rb]: This is wrong. assert_equal takes expected then actual. it was right before

File: test/models/game_test.rb [hunk 3]
```diff
@@ -92,10 +92,10 @@ def setup
 
     g.confirm
 
-    assert_equal @a.points, 1016, "Player A's points did not return as 1016 after first win"
-    assert_equal @b.points, 1016, "Player B's points did not return as 1016 after first win"
-    assert_equal @c.points, 984, "Player C's points did not return as 984 after first loss"
-    assert_equal @d.points, 984, "Player D's points did not return as 984 after first loss"
+    assert_equal @a.points, 1019.2, "Player A's points did not return as 1016 after first win"
```
Comment (vpontis) [inline test/models/game_test.rb]: why did the numbers change and the messages don't line up now

File: test/models/group_test.rb [hunk 4]
```diff
@@ -45,7 +45,7 @@ def setup
     game.confirm
     game.save!
 
-    assert @a.points_in(g) == 1000.0, 'Player group rating updated'
+    assert @a.points_in(g) == 1003.2, 'Player group rating updated'
```
Comment (vpontis) [inline test/models/group_test.rb:48]: activity bonus seems to make this more confusing then it needs to be

File: test/models/player_test.rb [hunk 5]
```diff
@@ -53,6 +53,19 @@ def setup
...
 
+  test "activity bonus" do
+    g = Game.new
+    g.winners = [players(:a), players(:b)]
+    g.losers = [players(:c), players(:d)]
+    g.created_at = 5.day.ago 
+    g.save
+    assert_equal players(:a).activity_bonus, 0.0, "Player A received activity bonus before confirmation"
+    g.confirm
+    assert g.confirmed? == true, "game was not confirmed"
```
Comment (vpontis) [inline test/models/player_test.rb]: you don't need to == true, you can just do g.confirmed?

File: app/helpers/games_helper.rb [hunk 6]
```diff
@@ -10,4 +10,7 @@ def this_week(games)
   def before_this_week(games)
     games.select {|g| g.created_at < 1.week.ago}
   end
-end
+
+  def this_month(games)
+  	games.select {|g| g.created_at > 1.month.ago}
```
Comment (vpontis) [inline app/helpers/games_helper.rb]: where is this used?

Comment (santhoshnarayan) [inline app/helpers/games_helper.rb]: tried using this originally for recent games instead of selecting, not used anymore

File: db/schema.rb (hunk 1 above)
Comment (santhoshnarayan) [inline db/schema.rb]: commited and deleted, but im not sure either

File: test/models/group_test.rb (hunk 4 above)
Comment (santhoshnarayan) [inline test/models/group_test.rb:48]: it adds 3.2 points for every game you've played in the last month up to 32 points

File: test/models/game_test.rb (hunk 2 above)
Comment (santhoshnarayan) [inline test/models/game_test.rb]: oh oops i have to fix this in a lot of places then, will do that now

**Pull request closed without merging**